packages/gestalt/src/VideoControls.js

```diff
@@ -101,11 +101,11 @@ function VideoControls({
     setShowFullscreenButton(
       typeof document !== 'undefined' &&
         (document.fullscreenEnabled ||
-          // $FlowFixMe[prop-missing]
+          // $FlowExpectedError[prop-missing]
```

Comment:
These fields are non-standard, so aren't expected to be part of the official Flow type for `document`